static/js/subs.js

```diff
@@ -859,7 +859,10 @@ $(function () {
 
     $(".subscriptions").on("click", "[data-dismiss]", function (e) {
         e.preventDefault();
-        show_subs_pane.nothing_selected();
+        // we want to make sure that the click is not just a simulated click.
```

Comment:
I'm going to expand this comment with much of the commit message content for clarity.